app/user_config.js

```diff
@@ -1,9 +1,12 @@
-const app = require('electron').app;
 const path = require('path');
+
+const app = require('electron').app;
```

Comment:
Just curious, is there some pattern we're going for in organizing these requires? Looks like maybe node stuff, then electron stuff, then our own stuff?

Reply:
Yep, that's generally how I like to do it. Node core, node modules, then our stuff. Eventually, I'd split our own stuff into stuff from other parts of the app, then stuff from the local dir.